example/index.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/no-multi-comp, react/prop-types */
```

Comment:
I think we should specify the prop-types.
I'm +1 in keeping `eslint-disable react/no-multi-comp` as additional meta-data, because as you say it will benefit new comers